### PR TITLE
DeletePriceModifierButton and sort icons fixes.

### PR DESCRIPTION
**Pull request by nicolad**
## Problem this Pull Request solves
image
***
image
***
## How has this been tested
image
***
image

### CODE REVIEW

File: assets/src/application/ui/display/icons/svgs/trash.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<svg
```
Comment (nicolad) [inline assets/src/application/ui/display/icons/svgs/trash.svg:1]: source: https://raw.githubusercontent.com/WordPress/dashicons/master/sources/svg/trash.svg

File: assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/buttons/PriceModifierActions.tsx [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
 const PriceModifierActions: React.FC<PriceModifierProps> = ({ index, price }) => {
 	return (
-		<div className='ee-price-modifier-actions' style={style}>
-			<AddPriceModifierButton key={'add'} index={index} />
-			{!price.isBasePrice && <DeletePriceModifierButton key={'delete'} price={price} />}
-		</div>
+		<Box className='ee-price-modifier-actions' display='flex'>
+			<AddPriceModifierButton index={index} key='add' />
+
+			<Divider orientation='vertical' borderColor='transparent' />
```
Comment (nicolad) [inline assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/buttons/PriceModifierActions.tsx:13]: this divider is quite handy

File: assets/src/application/ui/display/EntityListLegend/index.tsx [hunk 3]
```diff
@@ -9,10 +10,15 @@ const EntityListLegend: React.FC<EntityListLegendProps> = ({ legendConfig }) =>
...
+		// @ts-ignore
+		const term = (!is(String)(icon) && React.createElement(icon)) || (
+			<Icon aria-label={description} name={icon as IconName} svgSize={18} />
+		);
+
 		return {
 			className: 'ee-entity-list-legend-item',
 			description,
-			term: <Icon aria-label={description} name={icon as IconName} svgSize={18} />,
+			term,
```
Comment (nicolad) [inline assets/src/application/ui/display/EntityListLegend/index.tsx:21]: image

File: assets/src/application/ui/display/EntityListLegend/index.tsx [hunk 4]
```diff
@@ -9,10 +10,15 @@ const EntityListLegend: React.FC<EntityListLegendProps> = ({ legendConfig }) =>
 	const { icons, swatches } = legendConfig;
 
 	const iconsSource = icons.map(({ icon, description }) => {
+		// @ts-ignore
+		const term = (!is(String)(icon) && React.createElement(icon)) || (
```
Comment (nicolad) [inline assets/src/application/ui/display/EntityListLegend/index.tsx:14]: not the cleanest code I've written, but will have to keep this until will transition to SVGs

File: assets/src/application/ui/display/icons/svgs/sort.svg [hunk 5]
```diff
@@ -1,11 +1,12 @@
 <svg
 	aria-hidden="true"
+	class="ee-svg"
+	fill="currentColor"
```
Comment (tn3rb) [inline assets/src/application/ui/display/icons/svgs/sort.svg:4]: 👍

File: assets/src/application/ui/layout/entityList/filterBar/buttons/ToggleSortingButton.tsx [hunk 6]
```diff
@@ -20,16 +20,16 @@ const ToggleSortingButton: React.FC<ToggleSortingButtonProps> = ({
...
-			labelClassName={ 'ee-filter-bar__btn-wrap' }
-			labelPosition={ LabelPosition.BOTTOM_CENTER }
-			{ ...rest }
+			className={className}
+			icon={Sort}
+			id={id}
+			label={__('sorting')}
+			onClick={toggleSorting}
+			tooltip={tooltip}
+			tooltipProps={{ placement: 'top' }}
```
Comment (manzoorwanijk) [inline assets/src/application/ui/layout/entityList/filterBar/buttons/ToggleSortingButton.tsx:29]: Lets avoid object literals for props, they cause re-renders for being created everytime.

**Pull request merged**